tools/tests/compiler-plugins.js

```diff
@@ -103,8 +103,13 @@ selftest.define("compiler plugin caching - coffee", () => {
     });
 
     // First program built (web.browser) compiles everything.
-    cacheMatch('Ran (#1) on: ' + JSON.stringify(
-      ["/subdir/nested-root." + extension, "/top." + extension]));
+    cacheMatch('Ran (#1) on: ' + JSON.stringify([
+      // Though files in imports directories are compiled, they are marked
+      // as lazy so they will not be loaded unless imported.
+      "/imports/dotdot." + extension,
```

Comment:
This now fails on the stylus self test however: https://circleci.com/gh/meteor/meteor/1389